python/py-pytorch/Portfile

```diff
@@ -73,15 +80,37 @@ if {${name} ne ${subport}} {
         port:py${python.version}-yaml \
         port:py${python.version}-torchtext
     
+    depends_run-append \
+        port:py${python.version}-onnx
+
+    depends_test-append \
+                    port:py${python.version}-pytest
+
     if {${python.version} < 35} {
         depends_lib-append \
             port:py${python.version}-typing
     }
 
     post-fetch { system -W ${worksrcpath} "git submodule update --init --recursive" }
 
-    # ZSTD support broke with 1.2.0. check with future releases.
-    set t_build_env "USE_ROCM=ON USE_NCCL=ON USE_LMDB=ON USE_LITE_PROTO=ON USE_LEVELDB=ON USE_GFLAGS=ON USE_GLOG=ON USE_OPENCV=ON USE_OPENMP=ON USE_CUDA=OFF USE_ZSTD=OFF USE_ZMQ=ON USE_TBB=ON CMAKE_LIBRARY_PATH=${prefix}:${prefix}/lib/libomp LIBRARY_PATH=${prefix}:${prefix}/lib/libomp"
+    # ZSTD support broke with 1.2.0-1.3.1. Check with future releases.
+    set t_build_env "\
+        CMAKE_LIBRARY_PATH=${prefix}:${prefix}/lib/libomp\
+        LIBRARY_PATH=${prefix}:${prefix}/lib/libomp\
```

Comment:
@cjones051073 @stromnov @reneeotten 

Shouldn't this be like this?
> `CMAKE_LIBRARY_PATH=${prefix}/lib:${prefix}/lib/libomp`

See, e.g.:

https://github.com/macports/macports-ports/blob/3176016d5543c494042eb25578e0a192b10bf65e/python/py-pytorch/files/FindMKL-OMP.patch#L9-L10

https://github.com/macports/macports-ports/blob/4921de2a0759c13ea58f2947a34fa73c3efb229f/devel/cmake/Portfile#L137

It's obviously never caused a build issue, but still doesn't look correct.

Reply:
And the same question for `LIBRARY_PATH` too.

Reply:
Also a style/organization question:

Wouldn't it be preferable to stick all these definitions in `build.env` and `destroot.env`?

Reply:
Yes it looks like /lib is missing in a few places here. Please just add it..

Reply:
Done